### ISSUE TITLE
Add "Materialize  shader pack for Godot"

### ISSUE THREAD

**Issue by anderlli0053** [BoundingBoxSoftware/Materialize#36]
Add `Materialize  shader pack for Godot` to the official website, under Downloads section -> https://www.boundingboxsoftware.com/materialize/downloads.php

**Comment by Legend-SCR-Official** [comment]
Materialize has been abandoned for years, I don't think the developer is gonna come back to add another shader T-T
Am working on a recreation of Materialize though :D

**Comment by anderlli0053** [comment]
Yes, sadly. It's nearly 3 years since my previous post and nothing new happened, no new commits, no answers, ... nothing :| .
Now the only way forward is that someone forks the repo and work on the software themselves.

**Comment by enkvadrat** [comment]
https://github.com/maikramer/Materialize seems to have some activity
